### PR TITLE
Try to fix mxe (again)

### PR DESCRIPTION
**Pull request by elpaso**

### CODE REVIEW

**Comment by roya0045** [comment]
Hey it worked! Nice work on the follow up. Though I'm not sure why the logs were buggy in qgis.

File: .github/workflows/mxe.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 name: Windows cross build with MXE
 
-on: [push]
+on: [push, pull_request]
```
Comment (nyalldawson) [inline .github/workflows/mxe.yml:3]: good move!

**Review by nyalldawson — APPROVED** [review]

**Comment by nirvn** [comment]
@elpaso , our 3.12 branch seems to show MXE build broken there too, do we need to backport?

**Comment by elpaso** [comment]
@nirvn probably yes, I didn't check if it broke for the same reason (python dependency in WITH_QGIS_PROCESS)

**Comment by roya0045** [comment]
Welp now MXE is running out of space when getting dependencies...

**Comment by elpaso** [comment]
Maybe temporary, I'm experiencing various failures today from other CI systems too.

**Comment by roya0045** [comment]
@elpaso possible, usually there should more room and no big dependency was added recently.

**Pull request merged**